autogen_subdomain.go

```diff
@@ -0,0 +1,58 @@
+package api
+
+import (
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
+	"net/http"
+	"path"
+	"strconv"
+)
+
+type AutogenSubdomain struct {
```

Comment:
Should this be `AutogenSubdomains` to be consistent with the others and to match the REST endpoints `.../autogen_subdomains`?